<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>917</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="s_highLandOctLbl">
      <property name="title">
       <string>Terrain Properties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="1">
        <widget class="QSlider" name="s_lowLandNoiseFreqSld">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
         <property name="sliderPosition">
          <number>25</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QDoubleSpinBox" name="s_highlandNoiseFreqSpn">
         <property name="maximum">
          <double>3.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="s_highLandNoiseFreqLbl">
         <property name="text">
          <string>High Land Noise Frequency</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="s_lowLandOctSpn">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="s_lowLandOctSld">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="22" column="1">
        <widget class="QPushButton" name="s_genTerrainBtn">
         <property name="text">
          <string>Generate Terrain</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="s_lowLandOctLbl">
         <property name="text">
          <string>Low Land Noise Octaves</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>High Land Noise Octaves</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="s_highLandNoiseFreqSld">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="s_moutainNoiseFreqLbl">
         <property name="text">
          <string>Mountain Noise Frequency</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="s_lowLandNoiseFreqSpn">
         <property name="maximum">
          <double>3.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QDoubleSpinBox" name="s_mountainNoiseFreqSpn">
         <property name="maximum">
          <double>3.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QDoubleSpinBox" name="s_caveThreshSpn">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>0.480000000000000</double>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSlider" name="s_mountainNoiseSld">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="s_lowLandNoiseFreqLbl">
         <property name="text">
          <string>Low Land Noise Frequency</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="s_mountainNoiseOctLbl">
         <property name="text">
          <string>Mountain Noise Octaves</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QSlider" name="s_marchingCubesResSdl">
         <property name="maximum">
          <number>128</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <widget class="QPushButton" name="s_genPreviewBtn">
         <property name="text">
          <string>Generate Preview</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLabel" name="s_grassDenseLbl">
         <property name="text">
          <string>Grass Density</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="s_marchResLbl">
         <property name="text">
          <string>Marching Cubes Resolution</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSlider" name="s_caveAppBiasSld">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="s_caveAppBiasLbl">
         <property name="text">
          <string>Cave Attenuate Bias</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QLabel" name="s_terrainPreviewLbl">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="s_caveThresholdLbl">
         <property name="text">
          <string>Cave Threshhold</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QDoubleSpinBox" name="s_caveAppBiasSpn">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
       <item row="17" column="2">
        <widget class="QSpinBox" name="s_marchResSpn">
         <property name="maximum">
          <number>128</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSlider" name="s_caveThreshSld">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>48</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QSlider" name="s_grassDenceSld">
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="19" column="2">
        <widget class="QSpinBox" name="s_grassDenseSpn">
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1097</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>864</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>1104</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1104</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>913</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>1091</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_marchingCubesResSdl</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>s_marchResSpn</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>987</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>1093</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_marchResSpn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>s_marchingCubesResSdl</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1099</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>961</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_grassDenseSpn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>s_grassDenceSld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1104</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>1061</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_grassDenceSld</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>s_grassDenseSpn</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>1103</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
